tools/src/main/scala/com/nvidia/spark/rapids/tool/qualification/QualificationArgs.scala

```diff
@@ -45,8 +45,11 @@ Usage: java -cp rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*
           "eg: 100-oldest (for processing oldest 100 event logs)")
   val applicationName: ScallopOption[String] =
     opt[String](required = false,
-      descr = "Filter event logs whose application name matches exactly with input string" +
-          "i.e no regular expressions supported.")
+      descr = "Filter event logs whose application name matches " +
+          "exactly or is a substring of input string. Regular expressions not supported." +
+          "For filtering based on complement of application name, use ~APPLICATION_NAME. Filter " +
```

Comment:
the second setence Filter event logs based on complement... seems extra to me as you just said that.